src/basecamp_cohort_0.cairo

```diff
@@ -105,6 +115,66 @@ func student_register{syscall_ptr:felt*, pedersen_ptr: HashBuiltin*, range_check
     return ()
 end
 
+@external
+func set_starknet_basecamp_address{syscall_ptr:felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(address: felt):
+    only_moderator()
+    starknet_basecamp_address_.write(address)
+    return ()
+end
+
+@external
+func mint_basecamp_token{syscall_ptr:felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+):
```

Comment:
Student can mint a graduation token if they've completed the course, in addition to other checks as seen below. This calls the ERC721 contract's `mint` function and mints the token to the student (caller)